Client/Frontend/Strings.swift

```diff
@@ -725,6 +725,31 @@ extension String {
             value: "Sign Up and Log In",
             comment: "Describes an action on the sync onboarding page in our Onboarding screens. This string will be on a button so user can sign up or login directly in the onboarding.",
             lastUpdated: .v102)
+        public static let WallpaperSelectorTitle = MZLocalizedString(
+            "Onboarding.Wallpaper.Title.v105",
+            value: "Try a splash of color",
+            comment: "Title for the wallpaper onboarding modal displayed on top of the homepage. This describes to the user that they can choose different wallpapers.",
+            lastUpdated: .v105)
```

Comment:
Should we tag for v106? because the string freeze for v105 was already done

Reply:
Good point! Will do.